docs/ROADMAP.md

```diff
@@ -12,82 +12,97 @@ There is an urgent, global need for an official WHO App to help contain and miti
+#### Technical Features / Requirements
+
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=) ([#381](https://github.com/WorldHealthOrganization/app/issues/381))
+* Native look-and-feel, designed to present content on mobile screens ([#382](https://github.com/WorldHealthOrganization/app/issues/382))
+* Localized content translated to at least one other language besides English ([#383](https://github.com/WorldHealthOrganization/app/issues/383))
+* Offline functionality with default content baked into the app bundle ([#384](https://github.com/WorldHealthOrganization/app/issues/384))
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates ([#385](https://github.com/WorldHealthOrganization/app/issues/385))
+* Localized push notification capabilities ([#387](https://github.com/WorldHealthOrganization/app/issues/387))
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Instrumentation/analytics capabilities for understanding aggregated usage information to be used for product improvement, complying with all applicable laws & regulations and app policies ([#389](https://github.com/WorldHealthOrganization/app/issues/389))
+* _(Needs Discussion)_ Distribution pipeline using [fastlane](https://fastlane.tools/) for integration with WHO's existing app distribution process ([#318](https://github.com/WorldHealthOrganization/app/issues/318))
+* Full security review
+
+#### Features NOT Included
+
+* Triage survey advising users on actions to take based on their symptoms
+  * Currently investigating options to integrate, including COVID Check
+  * Ideally, this will be a fast follow-on
+* Translation into wide variety of languages
+  * Although we will shoot for as many of WHO's 6 official languages as possible, the requirement is only for English + one additional language
+  * We will want to thoroughly vet any translations that we include in a public release
+* Localization of content based on country
+  * We may address this technically anyway based on our solution to content localization, but it is not a requirement for v1.0
+
+#### Non-Technical Requirements
+
+* Content spec finalized
+* Design spec finalized
+* Legal deliverables
+  * Addendum to privacy policy for analytics
+  * Legal review, including ToS/EULA if required
+* Updated visual and text assets for app store listings
+
+### vNext backlogs
+
+This section is a major WORK IN PROGRESS and is not fully complete. Listing out very initial thoughts for now.
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/3
+
+#### Fast Follow-Ons
+
+Features that we will want in upcoming milestones quickly after v1.0 release:
+
+* Triage survey integration
+
+## Release Notes
```

Comment:
I don't think Release Notes should be in a roadmap.  They should be in CHANGELOG.md.

Reply:
copy that, thought it'd be easier for this, but I'll pull into a separate file